### ISSUE TITLE
[feat] Add the color properties of the taskbar button to the theme XAML

### ISSUE THREAD

**Issue by anzz1** [stnkl/EverythingToolbar#230]
Currently you cannot change the color scheme of the taskbar button, only the app window itself. Exposing this property in the xaml file would allow people to change the coloring (and maybe an option to choose an transparent brush instead of a solid one).

In Win7 the taskbar button currently looks like this, otherwise the app functions on Win7 absolutely fine (with the exception of "Replace start menu search" option obviously not working as it's targeting Win10).
![Taskbar](https://user-images.githubusercontent.com/13628128/158490374-2e318369-6cef-4122-a2aa-38bd949ec1fe.png)

**Comment by Wrong-Code** [comment]
I second this. Making the toolbar background transparent would adapt it automatically to any customization applied to the Windows taskbar (the snapshot below shows Windows 10 taskbar with one of the Stardock Curtains themes)

![image](https://user-images.githubusercontent.com/47647430/163552224-5e77824a-5021-4390-ba8d-6895c6722624.png)

**Comment by stnkl** [comment]
The background color of the search button should be transparent already as can be seen here: 
![grafik](https://user-images.githubusercontent.com/17520641/163552961-0462f4c6-be8b-4e1d-8ad7-189eff367d82.png)

@Wrong-Code Does the problem disappear if you uninstall/disable Stardock Curtains? I have no experience with that tool.

Also regarding @anzz1 I don't know if deskband transparency is supported on Windows 7. I would have to try it on a fresh installation maybe when I get the time. Then I could also try if there is an easy way to get the start menu replacement to work. But, to be honest, since Windows 7 reached EOL already I consider this pretty low priority at the moment.

**Comment by Wrong-Code** [comment]
@stnkl Yes, the problem disappears if I restore the standard Windows 10 theme. However I've just discovered that the issue is exactly the opposite: doing some experiments I enabled the semi-transparent toolbar, and from the two snapshots below it's clear that there are incompatibilities between your toolbar and the way Stardock Curtains skins the taskbar. Unfortunately I can't say which it is.

1st snapshot is with standard Windows 10 theme,

![image](https://user-images.githubusercontent.com/47647430/163604510-5dc8e264-7b0b-4f89-bdab-88e9516fa8ba.png)

2nd snapshot is with one of the provided Curtain's themes. The toolbar button is completely transparent and it draws its glyph on top of the content of the window behind the taskbar.

![image](https://user-images.githubusercontent.com/47647430/163604537-1438c397-7c4a-4360-83e7-5ddfea100b31.png)
